### PR TITLE
Updated Planned Time Off Page

### PR DESCRIPTION
**Pull request by yomiadetutu1**

### CODE REVIEW

File: operations/workplace/people/working-at-mattermost/paid-time-off/README.md [hunk 1]
```diff
@@ -2,43 +2,45 @@
...
 
 Note: If you are taking [Pregnancy](../leaves-of-absence/pregnancy-leave.md) or [Parental Leave](../leaves-of-absence/pregnancy-leave.md), please see [Leave Policies](../leaves-of-absence/).
 
+## Statutory Vacation Requirements
+There are statutory vacation entitlements in countries where Mattermost has an entity or EOR. At a minimum, staff must use their statutory vacation entitlement each year and it is the responsibility of managers to ensure that their direct reports are taking the required statutory minimum vacation. We understand there may be circumstances where, due to exceptional operational needs or personal issues, it isn’t possible to use up the full annual statutory vacation entitlement.  Exceptions may be granted for vacation carry over in those situations or where there is a statutory requirement to do so. Statutory vacation day carryover must be approved in advance by your manager, Team MLT, and the People Team. Staff members should work with their managers to discuss a plan for booking any accrued but unused statutory vacation.
```
Comment (lynn915) [inline operations/workplace/people/working-at-mattermost/paid-time-off/README.md]: There may be statutory vacation entitlements in countries where Mattermost has an entity or EOR.

File: operations/workplace/people/working-at-mattermost/paid-time-off/README.md [hunk 2]
```diff
@@ -2,43 +2,45 @@
...
-5. As long as you have entered your time off in BambooHR, and you have arranged for coverage while you are gone, you are able to take your vacation before it has been approved in BambooHR. More than 14 consecutive days off require manager approval in BambooHR prior to the vacation starting.
-6. Be [vacation-ready](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#vacation-ready): 
+1. Communicate your Paid Time Off (PTO) by booking time off in **Rippling > Time Off > Schedule Time Off**. You can also watch this [short video](https://drive.google.com/file/d/1AmDFbIHGsBTPKg-TTTVlO6gAxSuLDFv0/view?usp=drive_link) on how to book time off in Rippling. PTO covers any number of things - including vacations, stay-cations, moving days, etc. When submitting vacation requests in Rippling, select "Planned Time Off". PTO is in addition to official public holidays in your region of residence. In some countries where there are statutory vacation requirements, public holidays are a supplementary entitlement.
```
Comment (lynn915) [inline operations/workplace/people/working-at-mattermost/paid-time-off/README.md]: Communicate your Planned Time Off (PTO).

File: operations/workplace/people/working-at-mattermost/paid-time-off/README.md [hunk 3]
```diff
@@ -2,43 +2,45 @@
...
+2. For audit and compliance purposes, Time Off for EOR staff will also be recorded in Deel by the People team.
+3. Statutory public holidays for FT Staff in entity and EOR locations (Australia, Canada, Germany, Spain, Sweden, US, UK) are automatically listed in Rippling. *For all FT Staff outside of entity and EOR locations, please ensure to enter any statutory public holidays taken in your region of residence in Rippling > Time Off > Public Holiday to distinguish it from vacation time.*
```
Comment (lynn915) [inline operations/workplace/people/working-at-mattermost/paid-time-off/README.md]: please ensure to enter any public holidays in your region of residence in Rippling > Time Off > Public Holiday to distinguish it from PTO.

**Review by lynn915 — CHANGES_REQUESTED** [review]
I made a few notes for changes. Once those are updated, this is approved.

File: operations/workplace/people/working-at-mattermost/paid-time-off/README.md (hunk 1 above)
Comment (yomiadetutu1) [inline operations/workplace/people/working-at-mattermost/paid-time-off/README.md]:
```suggestion
There may be statutory vacation entitlements in countries where Mattermost has an entity or EOR. At a minimum, staff must use their statutory vacation entitlement each year and it is the responsibility of managers to ensure that their direct reports are taking the required statutory minimum vacation. We understand there may be circumstances where, due to exceptional operational needs or personal issues, it isn’t possible to use up the full annual statutory vacation entitlement.  Exceptions may be granted for vacation carry over in those situations or where there is a statutory requirement to do so. Statutory vacation day carryover must be approved in advance by your manager, Team MLT, and the People Team. Staff members should work with their managers to discuss a plan for booking any accrued but unused statutory vacation. 
```

File: operations/workplace/people/working-at-mattermost/paid-time-off/README.md [hunk 4]
```diff
@@ -2,43 +2,45 @@
...
-5. As long as you have entered your time off in BambooHR, and you have arranged for coverage while you are gone, you are able to take your vacation before it has been approved in BambooHR. More than 14 consecutive days off require manager approval in BambooHR prior to the vacation starting.
-6. Be [vacation-ready](https://handbook.mattermost.com/company/about-mattermost/list-of-terms#vacation-ready): 
+1. Communicate your Paid Time Off (PTO) by booking time off in **Rippling > Time Off > Schedule Time Off**. You can also watch this [short video](https://drive.google.com/file/d/1AmDFbIHGsBTPKg-TTTVlO6gAxSuLDFv0/view?usp=drive_link) on how to book time off in Rippling. PTO covers any number of things - including vacations, stay-cations, moving days, etc. When submitting vacation requests in Rippling, select "Planned Time Off". PTO is in addition to official public holidays in your region of residence. In some countries where there are statutory vacation requirements, public holidays are a supplementary entitlement.
```
Comment (yomiadetutu1) [inline operations/workplace/people/working-at-mattermost/paid-time-off/README.md]:
```suggestion
1. Communicate your Planned Time Off (PTO) by booking time off in **Rippling > Time Off > Schedule Time Off**. You can also watch this [short video](https://drive.google.com/file/d/1AmDFbIHGsBTPKg-TTTVlO6gAxSuLDFv0/view?usp=drive_link) on how to book time off in Rippling. PTO covers any number of things - including vacations, stay-cations, moving days, etc. When submitting vacation requests in Rippling, select "Planned Time Off". PTO is in addition to official public holidays in your region of residence. In some countries where there are statutory vacation requirements, public holidays are a supplementary entitlement.
```

File: operations/workplace/people/working-at-mattermost/paid-time-off/README.md [hunk 5]
```diff
@@ -2,43 +2,45 @@
...
+1. Communicate your Planned Time Off (PTO) by booking time off in **Rippling > Time Off > Schedule Time Off**. You can also watch this [short video](https://drive.google.com/file/d/1AmDFbIHGsBTPKg-TTTVlO6gAxSuLDFv0/view?usp=drive_link) on how to book time off in Rippling. PTO covers any number of things - including vacations, stay-cations, moving days, etc. When submitting vacation requests in Rippling, select "Planned Time Off". PTO is in addition to official public holidays in your region of residence. In some countries where there are statutory vacation requirements, public holidays are a supplementary entitlement.
+2. For audit and compliance purposes, Time Off for EOR staff will also be recorded in Deel by the People team.
+3. Statutory public holidays for FT Staff in entity and EOR locations (Australia, Canada, Germany, Spain, Sweden, US, UK) are automatically listed in Rippling. *For all FT Staff outside of entity and EOR locations, please ensure to enter any statutory public holidays taken in your region of residence in Rippling > Time Off > Public Holiday to distinguish it from vacation time.*
```
Comment (yomiadetutu1) [inline operations/workplace/people/working-at-mattermost/paid-time-off/README.md]:
```suggestion
3. Statutory public holidays for FT Staff in entity and EOR locations (Australia, Canada, Germany, Spain, Sweden, US, UK) are automatically listed in Rippling. *For all FT Staff outside of entity and EOR locations, please ensure to enter any statutory public holidays taken in your region of residence in Rippling > Time Off > Public Holiday to distinguish it from PTO.* 
```

**Review by yomiadetutu1 — COMMENTED** [review]
Updated with feedback

**Review by lynn915 — APPROVED** [review]
LGTM

**Review by lynn915 — APPROVED** [review]
Looks good to me

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**